mteb/abstasks/AbsTaskInstructionRetrieval.py

```diff
@@ -326,205 +326,176 @@ def load_data(self, **kwargs):
-                }
+        for k, v in scores_og.items():
+            overall_changed_scores[f"individual_original_{k}"] = v
+        for k, v in scores_changed.items():
+            overall_changed_scores[f"individual_changed_{k}"] = v
 
-                if self.do_length_ablation:
-                    keywords, short_instructions = (
-                        self.keywords[lang][split],
-                        self.short_instructions[lang][split],
-                    )
-                    scores_base[lang], results_base[lang] = self._evaluate_subset(
-                        retriever,
-                        corpus,
-                        queries,
-                        og_relevant_docs,
-                        defaultdict(str),
-                        top_ranked,
-                        lang,
-                        **kwargs,
-                    )
-                    scores_w_keywords = self._evaluate_subset(
-                        retriever,
-                        corpus,
-                        queries,
-                        og_relevant_docs,
-                        keywords,
-                        top_ranked,
-                        lang,
-                        **kwargs,
-                    )
-                    scores_w_short_instr = self._evaluate_subset(
-                        retriever,
-                        corpus,
-                        queries,
-                        og_relevant_docs,
-                        short_instructions,
-                        top_ranked,
-                        lang,
-                        **kwargs,
-                    )
-                    overall_changed_scores[lang]["length_ablation"] = {
-                        "keywords": scores_w_keywords,
-                        "short_instructions": scores_w_short_instr,
-                        "base": scores_base[lang],
-                    }
-        else:  # seems like these two can be combined into one (with the new lang="default")
-            lang = "default"
-            corpus, queries = self.corpus[split], self.queries[split]
-            og_relevant_docs, changed_relevant_docs = (
-                self.og_relevant_docs[split],
-                self.changed_relevant_docs[split],
-            )
-            og_instructions, changed_instructions = (
-                self.og_instructions[split],
-                self.changed_instructions[split],
+        if self.do_length_ablation:
+            keywords, short_instructions = (
+                keywords[split],
+                short_instructions[split],
             )
-            top_ranked = self.top_ranked[split]
-
-            scores_og, results_og = self._evaluate_subset(
+            scores_base, results_base = self._evaluate_subset(
                 retriever,
                 corpus,
                 queries,
                 og_relevant_docs,
-                og_instructions,
+                defaultdict(str),
```

Comment:
Why did we replace this with an empty dict?

Reply:
I didn't replace it; the diff might appear broken because I moved parts around. I copied this from the existing code, which already had the same structure https://github.com/embeddings-benchmark/mteb/blob/main/mteb/abstasks/AbsTaskInstructionRetrieval.py#L409

Reply:
Totally right, sorry.  Looks great.